proto/file/file.proto

```diff
@@ -1,5 +1,7 @@
 syntax = "proto3";
```

Comment:
Please change only the functions that you really changed!
It is impossible to read this file

Reply:
okay? fixed io guess.

Reply:
done